### PR TITLE
Committing basic authentication code

### PR DESCRIPTION
**Pull request by mtolrom**
Profile page and password encryption need work later.

### CODE REVIEW

File: views/static/registration.php [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+    $numOfKids = $_POST['numOfKids'];
+    $userRole = $_POST['userRole'];
+    $password = $_POST['password_1'];
+
+    $query = "SELECT * FROM Users WHERE email='$emailAddress'";
+    $result = $link->query($query);
+    $count = mysqli_num_rows($result);
+//3.1.2 If the posted values are equal to the database values, then session will be created for the user.
+    if ($count == 0) {
+        $query = "INSERT INTO Users(email, password, firstName, lastName, phoneNumber, notificationState, UserTypeId) VALUES ('$emailAddress', '$password', '$firstName', '$lastName', '$phoneNumber', 1, 1)";
```
Comment (mviencek) [inline views/static/registration.php:20]: are usertypeid and notificationstate meant to be hard coded?

File: nbproject/private/private.properties [hunk 2]
```diff
@@ -0,0 +1,8 @@
+browser.id=SL[/Browsers/FirefoxBrowser
+copy.src.files=false
```
Comment (mviencek) [inline nbproject/private/private.properties:2]: please add the nbproject folder to your gitignore

File: views/static/login.php [hunk 3]
```diff
@@ -1,17 +1,47 @@
+<?php
+
+require_once('connection.php');
+$link = Db::getInstance();
+
+//3. If the form is submitted or not.
+//3.1 If the form is submitted
+if (!isset($_SESSION['emailAddress'])) {
+//3.1.1 Assigning posted values to variables.
+    $emailAddress = $_POST['emailAddress'];
```
Comment (mviencek) [inline views/static/login.php:10]: youre not checking if your posts are set before checking for them

**Review by mviencek — COMMENTED** [review]
please take a look at the couple of comments I made.  Over all looks great

File: views/static/login.php (hunk 3 above)
Comment (mviencek) [reply views/static/login.php:10]: i meant to say checking if they are set before grabbing them

File: views/static/registration.php (hunk 1 above)
Comment (mtolrom) [reply views/static/registration.php:20]: This is for testing purpose. I will work on this as well as password encryption and user profile.

File: nbproject/private/private.properties (hunk 2 above)
Comment (mtolrom) [reply nbproject/private/private.properties:2]: ok

File: views/static/login.php (hunk 3 above)
Comment (mtolrom) [reply views/static/login.php:10]: Will work on that later. I would like to post the overall authentication architecture.

**Comment by mviencek** [comment]
Im ok with you fixing the stuff later.  I agree with you that this needs to get committed.

**Review by mviencek — APPROVED** [review]
Thank you for getting this to us so quickly.

**Review by cjshaw — APPROVED** [review]
some netbeans stuff leftover but i'll approve so we can move on.

**Pull request merged**